### DISCUSSION TITLE
09 PyTorch Model Deployment 6.1 Visualizing the speed vs. performance tradeoff hit error

### DISCUSSION THREAD

**Discussion by bubble114** [mrdbourke/pytorch-deep-learning#793] [Q&A] [answered]
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
[<ipython-input-46-7fda6fbe73db>](https://localhost:8080/#) in <cell line: 31>()
     29 
     30 # Save the figure
---> 31 plt.savefig("images/09-foodvision-mini-inference-speed-vs-performance.jpg")
     32 
     33 # Show the figure

7 frames
[/usr/local/lib/python3.10/dist-packages/PIL/Image.py](https://localhost:8080/#) in save(self, fp, format, **params)
   2426                 fp = builtins.open(filename, "r+b")
   2427             else:
-> 2428                 fp = builtins.open(filename, "w+b")
   2429 
   2430         try:

FileNotFoundError: [Errno 2] No such file or directory: 'images/09-foodvision-mini-inference-speed-vs-performance.jpg'

**Comment by bubble114** [comment, marked as answer]
add below line can fix
!mkdir images
plt.savefig("images/09-foodvision-mini-inference-speed-vs-performance.jpg")

**Reply by mrdbourke** [reply]
Thank you for this!

Fixed here: https://github.com/mrdbourke/pytorch-deep-learning/commit/c65584211b4cf6517974872d32c5d1b9e03c6af8
